VOID/fitness/threshold.py

```diff
@@ -74,6 +106,77 @@ def __call__(self, complex):
         return self.normalize(self.get_distances(complex).min() - self.threshold)
 
 
+class MinDistanceCationAnionFitness(ThresholdFitness):
+    PARSER_NAME = "min_catan_distance"
```

Comment:
I don't think I saw a parser or function with this name. Is it located elsewhere? (maybe I am misunderstanding what the parser is and does)